### PR TITLE
Version Increment to 1.3.3 OpenSearch release

### PR DESCRIPTION
**Pull request by prudhvigodithi**
Signed-off-by: pgodithi <pgodithi@amazon.com>

### Description
https://github.com/opensearch-project/opensearch-build/issues/2101

### Issues Resolved
https://github.com/opensearch-project/opensearch-build/issues/2101

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: reports-scheduler/build.gradle [hunk 1]
```diff
@@ -279,7 +279,7 @@ String bwcFilePath = "src/test/resources/bwc"
     testClusters {
         "${baseName}$i" {
             testDistribution = "ARCHIVE"
-            versions = ["7.10.2","1.3.2-SNAPSHOT"]
+            versions = ["7.10.2","1.3.3-SNAPSHOT"]
```
Comment (joshuali925) [inline reports-scheduler/build.gradle:282]: np. `opensearch_version` can be used to replace `"1.3.3-SNAPSHOT"`

**Review by joshuali925 — DISMISSED** [review]
lgtm, build needs job-scheduler 1.3.3

**Comment by prudhvigodithi** [comment]
Hey @joshuali925 I have just added the `opensearch-job-scheduler-1.3.3.0-SNAPSHOT.zip` file.

**Review by joshuali925 — APPROVED** [review]
thanks

**Review by ps48 — APPROVED** [review]

**Review by anirudha — APPROVED** [review]

**Pull request merged**